strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/exception/SizeFormatValidationException.java

```diff
@@ -0,0 +1,10 @@
+package org.carlspring.strongbox.exception;
+
+public class SizeFormatValidationException extends RuntimeException {
```

Comment:
Cosmetic: Braces on new lines please. :)
```suggestion
public class SizeFormatValidationException extends RuntimeException
{
```